backend/src/spells.py

```
@@ -279,6 +281,48 @@ def sanitize_uploaded_schema(input_schema: FeedbackSchema) -> FeedbackSchema:
+    result_dirs = sorted(
+        d
+        for d in feedback_path.iterdir()
+        if d.is_dir()
+        and (parsed := _try_parse_date(d.name)) is not None
+        and parsed >= since_date
+    )
+
+    if not result_dirs:
+        return None
+
+    tmp = tempfile.NamedTemporaryFile(suffix=".tar.gz", delete=False)
```

Pylint (4.0.6)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)